helm/hpcc/values.yaml

```diff
@@ -300,14 +301,18 @@ sasha:
     #throttle: 0 # throttle ratio (0-99, 0 no throttling, 50 is half speed)
 
   dfurecovery-archiver:
-    #disabled: true
+    disabled: true
+    service:
+      servicePort: 8877
     #limit: 20 # threshold number of DFU recovery items before archiving starts (0 disables)
     #cutoff: 4 # minimum DFU recovery item age to archive (days)
     #interval: 12 # minimum interval between running DFU recovery archiver(in hours, 0 disables)
     #at: "* * * * *" # schedule to run DFU recovery archiver (cron format)
 
   file-expiry:
-    #disabled: true
+    disabled: true
```

Comment:
I assume not intended, just disabled while testing?

Reply:
reverted